devfiles/java-mysql-theia-kubernetes.yaml

```diff
@@ -0,0 +1,113 @@
+---
+displayName: Java and MySQL with Theia IDE on Kubernetes
+description: Multi Container Workspace - Java with MySQL database in Theia IDE
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+name: java-mysql-theia-kubernetes
+components:
+  -
+    type: cheEditor
+    id: eclipse/che-theia/next
+  -
+    type: chePlugin
+    id: eclipse/che-machine-exec-plugin/0.0.1
+  -
+    type: dockerimage
```

Comment:
I used `dockerimage` component because it is simpler than `kubernetes`, but original Che 7 stack contains kubernetes list (actually because there was no an ability to use any other recipe type because of multiple dockerimages).

With `kubernetes` component it would look like the following:
```yaml
  -
    type: kubernetes
    alias: web
    referenceContent: |
      kind: List
      items:
      -
        apiVersion: v1
        kind: Pod
        metadata:
          name: web
        spec:
          containers:
          -
            image: registry.centos.org/che-stacks/centos-jdk8
            name: dev
            ports:
            - containerPort: 8080
              protocol: TCP
            resources:
              limits:
                memory: 512Mi
            volumeMounts:
            - name: projects
              mountPath: /projects
          volumes:
          - name: projects
            persistentVolumeClaim:
              claimName: projects
      -
        apiVersion: v1
        kind: PersistentVolumeClaim
        metadata:
          name: projects
          labels:
            app.kubernetes.io/name: employee-manager
            app.kubernetes.io/component: nodejs-app
        spec:
          accessModes:
           - ReadWriteOnce
          resources:
            requests:
              storage: 1Gi
```
vs dockerimage
```yaml
    type: dockerimage
    alias: web
    dockerimage: registry.centos.org/che-stacks/centos-jdk8
    memoryLimit: 512Mi
    mountSources: true
    endpoints:
      - name: tomcat8
        port: 8080
````
Just wanted to ask if everybody is OK to use `dockerimage` instead of `kubernetes` whether it is possible.
@skabashnyuk @l0rd WDYT?

Reply:
For this particular case, it makes sense to leave kubernetes components since it's named as `java-mysql-theia-kubernetes`, but in other devfiles, I'm going to use dockerimage where it is possible to not `kuberentes` is mentioned in description or name.